docs/source/API/containers/ScatterView.md

```diff
@@ -23,53 +28,55 @@ Kokkos::Experimental::contribute(results, scatter);
 ```c++
 
 template <typename DataType
-        ,int Op
+        ,int Operation
         ,typename ExecSpace
         ,typename Layout
         ,int contribution
         >
 class ScatterView<DataType
                   ,Layout
                   ,ExecSpace
-                  ,Op
+                  ,Operation
                   ,{ScatterNonDuplicated,ScatterDuplicated}
                   ,contribution>
 {
 public:
  typedef Kokkos::View<DataType, Layout, ExecSpace> original_view_type;
  typedef typename original_view_type::value_type original_value_type;
  typedef typename original_view_type::reference_type original_reference_type;
- friend class ScatterAccess<DataType, Op, ExecSpace, Layout, {ScatterNonDuplicated,ScatterDuplicated}, contribution, ScatterNonAtomic>;
- friend class ScatterAccess<DataType, Op, ExecSpace, Layout, {ScatterNonDuplicated,ScatterDuplicated}, contribution, ScatterAtomic>;
+ friend class ScatterAccess<DataType, Operation, ExecSpace, Layout, {ScatterNonDuplicated,ScatterDuplicated}, contribution, ScatterNonAtomic>;
+ friend class ScatterAccess<DataType, Operation, ExecSpace, Layout, {ScatterNonDuplicated,ScatterDuplicated}, contribution, ScatterAtomic>;
+ []:# (TODO: Deprecate types requiring `Kokkos::Impl..` )
```

Comment:
Did you mean to commit this line?

Reply:
Do you mean the comment line, `[]:# (TODO: Deprecate types requiring `Kokkos::Impl..` )`?  If yes, that was my initial intent, but I can remove it.

Reply:
No I mean "commit".  Are you sure about the syntax?  The line did show on the rendered diff